src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android/Xamarin.Android.Bindings.ClassParse.targets

```diff
@@ -18,6 +18,12 @@ This file is only used by binding projects.
       Inputs="@(EmbeddedJar);@(EmbeddedReferenceJar);@(InputJar);@(ReferenceJar);@(_AndroidMSBuildAllProjects)"
       Outputs="$(ApiOutputFile)">
 
+    <PropertyGroup>
+      <!-- Allow $(_BindingsToolsLocation) to override where to find class-parse/generator -->
+      <_BindingsToolsLocation Condition=" '$(_BindingsToolsLocation)' == '' ">$(MonoAndroidToolsDirectory)</_BindingsToolsLocation>
+      <_BindingsToolsLocation Condition=" '$(UsingAndroidNETSdk)' != 'true' ">$(MonoAndroidToolsDirectory)</_BindingsToolsLocation>
```

Comment:
Are the conditions here trying to force `$(MonoAndroidToolsDirectory)` to be used in classic Xamarin.Android?

You could maybe just remove the second line and we don't worry about it.

Reply:
Yeah, I think so.  I think this has been sitting around since before we removed Classic from `main`.